### PR TITLE
Check attempts thresholds

### LINKED ISSUE #11: Implement `checkAttemptsThresholds` function
See [the reference](https://github.com/covid19cz/erouska-backend/wiki/Firebase-Functions#checkattemptsthresholds).

### PR DESCRIPTION
**Pull request by jendakol**
Closes #11

### CODE REVIEW

File: internal/functions/checkattemptsthresholds/check-attempts-thresholds.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	}
+
+	logger.Debugf("Handling CheckAttemptsThresholds request: %+v", request)
+
+	var date string
+	if request.Date == 0 {
+		date = utils.GetTimeNow().Format("20060102")
+	} else {
+		date = strconv.Itoa(request.Date)
+	}
```
Comment (marekaf) [inline internal/functions/checkattemptsthresholds/check-attempts-thresholds.go:60]: can we hide this in some internal/utils function?

File: internal/functions/checkattemptsthresholds/check-attempts-thresholds.go [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+	w.Header().Set("Content-Type", "application/json")
+	_, err = w.Write(js)
+	if err != nil {
+		response := fmt.Sprintf("Error: %v", err)
+		logger.Error(err)
+		http.Error(w, response, http.StatusInternalServerError)
+		return
+	}
```
Comment (marekaf) [inline internal/functions/checkattemptsthresholds/check-attempts-thresholds.go:96]: aren't we always setting application/json and handling the error the same way everywhere? we could write a wrapper function around this for reuse :D

**Review by marekaf — APPROVED** [review]
just minor feedback, feel free to merge:)

File: internal/functions/checkattemptsthresholds/check-attempts-thresholds.go (hunk 1 above)
Comment (jendakol) [reply internal/functions/checkattemptsthresholds/check-attempts-thresholds.go:60]: Good point. Will solve in following PR.

File: internal/functions/checkattemptsthresholds/check-attempts-thresholds.go (hunk 2 above)
Comment (jendakol) [reply internal/functions/checkattemptsthresholds/check-attempts-thresholds.go:96]: Good point. Will solve in following PR.

File: internal/functions/checkattemptsthresholds/check-attempts-thresholds.go (hunk 1 above)
Comment (jendakol) [reply internal/functions/checkattemptsthresholds/check-attempts-thresholds.go:60]: Well... maybe not in this case :smile: I'm not actually sure how to do that - let'S discuss in person.

**Pull request merged**